runtime/src/main/java/org/corfudb/runtime/view/stream/StreamAddressSpace.java

```diff
@@ -141,7 +141,10 @@ private long getFirst() {
      * @param address address to add.
      */
     public void addAddress(long address) {
-        if (address <= this.trimMark || Address.nonAddress(address)) {
```

Comment:
should we log instead so we know when it is happening and the address that appears in the trim mark to help debug?